src/views/home/PlayerHome/components/activities/NewsActivity.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <SmartLink :link="link">
+    <div class="page-content card">
+      <div class="container">
+        <div class="row justify-content-between">
+          <div class="col p-0">
+            <!-- Note: Any space between these tags will misalign the Type tag. -->
+            <div :style="{ color: typeColor(article) }">
+              <b>{{ formattedType(article) }}</b>
+            </div>
+          </div>
+          <div class="col p-0" style="text-align:right">
+            <div style="opacity: 0.5; font-weight: bold;">
+              {{ date }}
+            </div>
+          </div>
+        </div>
+
+        <h3 class="card-title" v-if="article.title">{{ article.title }}</h3>
+      </div>
+    </div>
+  </SmartLink>
+</template>
+<script lang="ts">
+  import { Component, Prop, Vue } from 'vue-property-decorator';
+  import SmartLink from '@/components/Common/SmartLink.vue';
+  import Utils from '@/utils/utils';
+  import {NewsItem as NewsItemType, BlogItem, NotificationType} from '@/types/common-types';
+  import MessageThread from '@/views/feed/ActivityFeed/components/MessageThread.vue';
+
+  @Component({
+    components: { SmartLink, MessageThread },
+  })
+  export default class NewsActivity extends Vue {
+    @Prop({ required: true }) readonly article!: NewsItemType|BlogItem;
+
+    private get link() {
+      return `/news/${this.article.nid}`;
+    }
+
+    private get commentCount() {
+      return this.article.type === NotificationType.NEWS ?
+        this.article.commentcount
+        : this.article.comments.length;
+    }
+
+    private get date() {
+      return new Date(this.article.created).toLocaleDateString(undefined, {
+        year: 'numeric',
+        month: 'short',
+        day: 'numeric'
+      });
+    }
+
+    private formattedType = Utils.formattedType;
+
+    private strippedBody = Utils.strippedBody;
+
+    private typeColor = Utils.typeColor;
+  }
+</script>
+
+<style lang="scss" scoped>
+  @import '@/styles/global.scss';
+
+  ::v-deep .card-body {
+    padding: 11.25px !important;
+  }
+
+  .card-title {
+    font-size: 1rem;
+    font-weight: bold;
+    margin: 0.1rem 0;
```

Comment:
I would replace the `margin` rule with `margin-top: 0.1rem` and `margin-bottom: 0.1rem`. This is in addition to adding the `row` class as mentioned above.